filer/private/sass/components/_filter-files.scss

```diff
@@ -48,6 +41,8 @@
         }
         &.open + .dropdown-menu-checkboxes {
             display: block;
+            width: 100%;
+            transition: width .2s;
```

Comment:
ideally also add this as a variable like: https://github.com/aldryn/aldryn-boilerplate-bootstrap3/blob/master/private/sass/settings/_custom.scss#L7